content/docs/command-reference/exp/rename.md

```diff
@@ -0,0 +1,80 @@
+# exp rename
+
+Rename a specific experiment from the <abbr>project</abbr>.
+
```

Comment:
cc @dberenbaum

We should probably add an info admon that this is a new command supported as of version 3.x.x
What's our convention here? I couldn't find an example

Reply:
We generally don't do this. I'm not against it but it will definitely generate opinions, so let's not let it block this PR.

Reply:
P.s. as a user it would be nice to have this in all docs (in my opinion). But like @dberenbaum said, this is not the standard, so I haven't added it here.